tracer/build/_build/Build.Steps.cs

```diff
@@ -658,17 +704,25 @@ partial class Build
         .After(CompileManagedUnitTests)
         .Executes(() =>
         {
-            var testProjects = TracerDirectory.GlobFiles("test/**/*.Tests.csproj")
+            var projectName = string.IsNullOrEmpty(UnitTestsProjectName) ? "test/**/*.Tests.csproj" : $"test/**/{UnitTestsProjectName}";
```

Comment:
Is this necessary? Can we leave it so we still _compile_ all the unit tests here, and just use the filter to avoid running them? I'm aware we already have a _lot_ of parameters 😬

Reply:
as requested in dec67549